### PR TITLE
updating az and test

### PR DESCRIPTION
**Pull request by zikalino**
- az should load readme.cli.md file as well
- adding config for autorest.test (test generation plugin)

### CODE REVIEW

File: core/resources/plugin-test.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Default Configuration - test
+
+``` yaml $(test)
```
Comment (fearthecowboy) [inline core/resources/plugin-test.md]: `test` is a reaaaaaly generic word. I'm pretty sure I have things that use `--test` which would trigger that. 

can we find a better switch for that?

Comment (zikalino) [reply core/resources/plugin-test.md]: ok

Comment (daviwil) [reply core/resources/plugin-test.md:3]: Circling back on this, what kind of tests is this generator generating?  Maybe I can help come up with a more specific name?

**Comment by daviwil** [comment]
There hasn't been any activity on this PR for a while so I'll go ahead and close it.  @zikalino, if you want to continue the discussion I'll be happy to reopen.

**Pull request closed without merging**